utils/crm_rpmcheck.py

```diff
@@ -23,6 +23,11 @@ def package_data(pkg):
     """
     Gathers version and release information about a package.
     """
+    if os.path.isfile('/bin/ansible'):
```

Comment:
Should use `shutil.which` instead, as it works better with environ `PATH` and also checks for 'x' permission.

Reply:
My idea was to make the code uniform. So that it looked like the rest. Like 
`if os.path.isfile('/bin/rpm'):`
and
`if os.path.isfile('/usr/bin/dpkg'):`
look all the same.
I could use the `shutil.which` indeed, but that would logically require refactoring the `/bin/rpm` and `/bin/dpkg` cases.

Reply:
So this is a good time to refactor them.